src/main/java/com/minecolonies/util/StructureWrapper.java

```diff
@@ -148,69 +129,49 @@ private static InputStream getStream(@NotNull ResourceLocation res)
     }
 
     /**
-     * Load a schematic into this world
+     * Load a structure into this world
      * and place it in the right position and rotation.
      *
      * @param worldObj  the world to load it in
-     * @param name      the schematics name
+     * @param name      the structures name
      * @param pos       coordinates
      * @param rotations number of times rotated
      */
-    public static void loadAndPlaceSchematicWithRotation(World worldObj, @NotNull String name, @NotNull BlockPos pos, int rotations)
+    public static void loadAndPlaceStructureWithRotation(World worldObj, @NotNull String name, @NotNull BlockPos pos, int rotations)
     {
         try
         {
-            @NotNull SchematicWrapper schematic = new SchematicWrapper(worldObj, name);
-            schematic.rotate(rotations);
-            schematic.placeSchematic(pos);
+            @NotNull StructureWrapper structureWrapper = new StructureWrapper(worldObj, name);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'structureWrapper' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)